docs/resources/aws_transit_gateway_multicast_domain.md

```diff
@@ -0,0 +1,85 @@
+---
+title: About the aws_transit_gateway_multicast_domain Resource
+platform: aws
+---
+
+# aws\_transit\_gateway\_multicast\_domain
+
+Use the `aws_transit_gateway_multicast_domain` InSpec audit resource to test properties of a single specific AWS EC2 TransitGatewayMulticastDomain.
+
+The AWS::EC2::TransitGatewayMulticastDomain resource creates a multicast domain using the specified transit gateway.
+
+## Syntax
+
+Ensure that transit gateway connect exists.
+
+    describe aws_transit_gateway_connect(transit_gateway_attachment_ids: 'tgw-attach-1234567890') do
+      it { should exits }
+    end
+
+## Parameters
+
+`transit_gateway_attachment_ids` _(required)_
+
+The transit_gateway_attachment_ids is the ID of the transit gateway multicast domain.
+
+For additional information, see the [AWS documentation on AWS EC2 TransitGatewayConnect](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-transitgatewayconnect.html).
+
+## Properties
+
+| Property | Description|
+| --- | --- |
+| transit_gateway_multicast_domain_id | The ID of the transit gateway multicast domain. |
+| transit_gateway_id | The ID of the transit gateway. |
+| transit_gateway_multicast_domain_arn | The Amazon Resource Name (ARN) of the transit gateway multicast domain. |
+| owner_id | The ID of the AWS account that owns the transit gateway multiicast domain. |
+| options (igmpv_2_support) | The options for the transit gateway multicast domain. Indicates whether Internet Group Management Protocol (IGMP) version 2 is turned on for the transit gateway multicast domain. |
+| options (static_sources_support) | The options for the transit gateway multicast domain. Indicates whether support for statically configuring transit gateway multicast group sources is turned on. |
+| options (auto_accept_shared_associations) | The options for the transit gateway multicast domain. Indicates whether to automatically cross-account subnet associations that are associated with the transit gateway multicast domain. |
+| state | The state of the transit gateway multicast domain. |
+| creation_time | The time the transit gateway multicast domain was created. |
+| tags | The tags for the transit gateway multicast domain. |
+
+## Examples
+
+### Ensure a transit gateway multicast domain id is available.
+    describe aws_transit_gateway_connect(transit_gateway_attachment_ids: 'tgw-attach-1234567890') do
```

Comment:
```suggestion
    describe aws_transit_gateway_connect(transit_gateway_attachment_id: 'tgw-attach-1234567890') do
```
There are multiple usages we can change that

Reply:
Done